### PR TITLE
Concurrent transaction execution skeleton

### PR DESCRIPTION
**Pull request by pattyshack**
I'll expose the concurrent execution option in a follow up PR.

Issue: #3547

### CODE REVIEW

File: engine/execution/computation/computer/computer.go [hunk 1]
```diff
@@ -425,6 +476,11 @@ func (e *blockComputer) executeTransactionInternal(
 		return txn, err
 	}
 
+	err = txn.Validate()
```
Comment (zhangchiqing) [inline engine/execution/computation/computer/computer.go:479]: why validating multiple times?

Comment (pattyshack) [reply engine/execution/computation/computer/computer.go:479]: i've added some comments to clarify this.

**Review by pattyshack — COMMENTED** [review]
github is acting strangely.  i've push the suggested change upstream, but it's not showing up in the PR ...

https://github.com/onflow/flow-go/blob/patrick/concurrent-executor/engine/execution/computation/computer/computer.go#L480

**Comment by pattyshack** [comment]
hmm.  let me close this PR and create a new one

**Pull request closed without merging**